src/transforms.jl

```diff
@@ -215,6 +215,7 @@ include("transforms/select.jl")
 include("transforms/filter.jl")
 include("transforms/rename.jl")
 include("transforms/coalesce.jl")
+include("transforms/replace.jl")
```

Comment:
Same here.

Reply:
OK

Reply:
Made in [9c5c293](https://github.com/JuliaML/TableTransforms.jl/pull/42/commits/9c5c293c10346a2cb1eddf7f2bf11923779906de)